mautic/mautic
# Add contact owner message to duplicate message

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ False]
| New feature/enhancement? (use the a.x branch)      | [ True]
| Deprecations?                          | [ None]
| BC breaks? (use the c.x branch)        | [None ]
| Automated tests included?              | [ No] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 



#### Description:
I added Mautic owner within the message that appear when user try to add contact with email, that already exists in database. 



#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Open contacts.
3. Add new contact, remember the email.
4. Try to add new contact using the same email.
5. Message should appear
